dev/DynamicDependency/M.AM.Converters.h

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+namespace Microsoft::ApplicationModel::DynamicDependency
+{
+    inline winrt::Windows::ApplicationModel::PackageVersion ToVersion(PACKAGE_VERSION const& from)
+    {
+        winrt::Windows::ApplicationModel::PackageVersion to{};
+        to.Major = from.Major;
+        to.Minor = from.Minor;
+        to.Build = from.Build;
+        to.Revision = from.Revision;
+        return to;
+    }
+
+    inline PACKAGE_VERSION ToVersion(winrt::Windows::ApplicationModel::PackageVersion const& from)
+    {
+        PACKAGE_VERSION to{};
+        to.Major = from.Major;
+        to.Minor = from.Minor;
+        to.Build = from.Build;
+        to.Revision = from.Revision;
+        return to;
+    }
+
+    inline MddPackageDependencyProcessorArchitectures ToArchitectures(winrt::Microsoft::ApplicationModel::DynamicDependency::PackageDependencyProcessorArchitectures const& architectures)
+    {
+        static_assert(static_cast<int32_t>(winrt::Microsoft::ApplicationModel::DynamicDependency::PackageDependencyProcessorArchitectures::None) == static_cast<int32_t>(MddPackageDependencyProcessorArchitectures::None), "WinRT/MddPackageDependencyProcessorArchitectures::None mismatch");
+        static_assert(static_cast<int32_t>(winrt::Microsoft::ApplicationModel::DynamicDependency::PackageDependencyProcessorArchitectures::Neutral) == static_cast<int32_t>(MddPackageDependencyProcessorArchitectures::Neutral), "WinRT/MddPackageDependencyProcessorArchitectures::Neutral mismatch");
+        static_assert(static_cast<int32_t>(winrt::Microsoft::ApplicationModel::DynamicDependency::PackageDependencyProcessorArchitectures::X86) == static_cast<int32_t>(MddPackageDependencyProcessorArchitectures::X86), "WinRT/MddPackageDependencyProcessorArchitectures::X86 mismatch");
+        static_assert(static_cast<int32_t>(winrt::Microsoft::ApplicationModel::DynamicDependency::PackageDependencyProcessorArchitectures::X64) == static_cast<int32_t>(MddPackageDependencyProcessorArchitectures::X64), "WinRT/MddPackageDependencyProcessorArchitectures::X64 mismatch");
+        static_assert(static_cast<int32_t>(winrt::Microsoft::ApplicationModel::DynamicDependency::PackageDependencyProcessorArchitectures::Arm) == static_cast<int32_t>(MddPackageDependencyProcessorArchitectures::Arm), "WinRT/MddPackageDependencyProcessorArchitectures::Arm mismatch");
```

Comment:
fyi - we are probably not going to support this architecture

Reply:
Fine by me. If the decision comes down I'll make a PR update. More important will be excising the ARM configurations from *.vcxproj (ugh)